e2e-tests/package.json

```diff
@@ -7,7 +7,7 @@
     "test": "playwright test"
   },
   "dependencies": {
-    "@playwright/test": "1.20.0",
+    "@playwright/test": "1.25.2",
```

Comment:
Here is the commit from the UI when we updated Playwright to `^1.21.x`, which added native ESM support when running TypeScript tests: https://github.com/NASA-AMMOS/aerie-ui/commit/4734ecf29aa1497e35063ce563e145339ddaecd8